### DISCUSSION TITLE
Issue in Breeze Environment

### DISCUSSION THREAD

**Discussion by Megha-Sai-Sree-PINNAKA** [apache/airflow#24377] [Q&A]
I have setup Breeze env in AWS ec2 Instance using the command :  ./breeze setup-autocomplete
After that when I gave the command :  ./breeze start-airflow -> I am getting the following error:
Error response from daemon: dockerfile parse error line 36: unknown instructions: FUNCTION
Error: No such image: ghrc.io/apache/airflow/main/ci/python3.7:latest
Error 1 returned

Can anyone please provide insights in sorting this issue.

**Comment by potiuk** [comment]
I think you use non-main version. Breeze is a developmemt environment and works best in "main" branch. I think the problem you have that you have not run `breeze build-image` first - this has been handled in the latest version - so you shouo not have to run `build-image` first, but if you use latest version it should happen automatically.

So to summarise:
1) get latest version (from main) and repeat
or
2) run `breeze build-image` first.

**Reply by Megha-Sai-Sree-PINNAKA** [reply]
I tried giving the command breeze build-image but then getting the following error:
Error response from daemon: dockerfile parse error line 36: unknown instruction: FUNCTION
Error when building image! Image build:3.7

Also I am on main branch itself.

**Reply by potiuk** [reply]
What is the environment you have? Seems like you miss buildkit in your docker installation (maybe the version of docker is too old or maybe you have some "dockerlike" replacement without installed buildkit extension). Buildkit is necessary to get breeze up and running as of Airlfow 2.3.* because we use newer featuers of Dockerfile specicifcation 

Look at the top of the Dockerfile:

```
# syntax=docker/dockerfile:1.4
```

Your "docker" command (wherever it comes from) must support this dockerfile syntax version

**Reply by Megha-Sai-Sree-PINNAKA** [reply]
Dockerfile and Dockerfile.ci both has # syntax=docker/dockerfile:1.4 as its first line.
I am working on AWS ec2 instance - Linux.
I tried giving DOCKER_BUILDKIT=1 and now I see a different error as follows on giving the command breeze build-image:

info There appears to be trouble with your network connection.Retrying...
Error when running yarn install:
error An unexpected error occurred: "https://registry.yarnpkg.com/lodash/-/lodash-4.17.21.tgz:ETIMEOUT".
executor failed running [/bin/bash -o pipefail -o errexit -o nounset -o nolog -c bash /scripts/docker/prepare_node-modules.sh]:
exit code:1
Error when building image: Image build: 3.7

I have these versions in my machine:
yarn version: 1.22.19
node version: v16.15.1
npm version: 8.11.0
docker version: 20.10.13
docker-compose version: v2.3.3
breeze version: 0.9.0

I tried giving :
- yarn add <package name> --network-timeout=100000
- deleting the node modules
- restarting docker
- breeze stop, breeze cleanup and then breeze build-image again

Seems like none of them worked, the error is still persisting.

**Reply by potiuk** [reply]
> info There appears to be trouble with your network connection.Retrying...

Have you tried to see if registry.yarnpkg.com is reachable? My guess you have a firewall that blocks it. Check with your company admin.,
